homeassistant/components/sensor/gearbest.py

```diff
@@ -0,0 +1,122 @@
+"""
+Parse prices of a item from gearbest.
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.gearbest/
+"""
+import logging
+import asyncio
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['gearbest_parser==1.0.4']
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ITEMS = 'items'
+CONF_CURRENCY = 'currency'
+
+ICON = 'mdi:coin'
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=2*60*60)  #2h * 60min * 60sec
+
+_ITEM_SCHEMA = vol.Schema({
+    vol.Optional("url"): cv.string,
+    vol.Optional("id"): cv.string,
```

Comment:
Not all items can be optional, at least something should be required or else how can one represent an item?